cdc/cdc/processor/pipeline/sorter.go

```diff
@@ -188,6 +212,7 @@ func (n *sorterNode) TryHandleDataMessage(ctx context.Context, msg pipeline.Mess
 					zap.Uint64("resolvedTs", resolvedTs),
 					zap.Uint64("oldResolvedTs", oldResolvedTs))
 			}
+			atomic.StoreUint64(&n.resolvedTs, rawKV.CRTs)
```

Comment:
Why ? The previous `oldResolvedTs := atomic.SwapUint64(&n.resolvedTs, resolvedTs)` should have done the assignment.

Reply:
Removed, thanks! PTAL~